### DISCUSSION TITLE
Hello I'm new to github crypto in general and I need help

### DISCUSSION THREAD

**Discussion by thebrazilbilly** [community/community#66080] [New to GitHub]
I am new to crypto not quite sure what I'm doing but I have a couple accounts with addresses and amounts in them but I'm not quite sure how to swap them or convert them or get them to show up on my metamask a
pp but they are on my metamask profile but one shows up as a watch account and I cannot figure out how to get it into my accounts list if someone does reply to this I have no idea if you replied or not I don't know if this is allowed but my email is the [redacted due to PII]

**Comment by queenofcorgis** [comment]
Hi @thebrazilbilly,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on crypto. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. GitHub is a software hosting service, we host millions of projects, so many projects/companies have software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. 

Best of luck!
